proposals/0035-linalg-matrix.md

```diff
@@ -1145,6 +1250,60 @@ a bias vector added to the result.
 
 > Note for this operation the matrix can be of any scope.
 
+```llvm
+declare void @dx.op.matrixAccumulateToDescriptor(
+  immarg i32,            ; opcode
+  %dx.types.MatrixRef *, ; matrix
+  %dx.types.Handle *,    ; RWByteAddressBuffer
+  i32,                   ; Offset
+  i32,                   ; Stride
+  i32                    ; matrix layout
+  )
+```
+
+Accumulates a matrix to a RWByteAddressBuffer at a specified offset. This
+operation is only available for matrices with `MatrixUse::Accumulator`. The
+matrix data is added to the existing data in the buffer. The matrix component
+data is converted to the target arithmetic or packed data type if the data types
+do not match, then added to the existing data in memory. If any destination
+address is out of bounds the entire accumulate operation is a no-op.
+
+```llvm
+declare void @dx.op.matrixAccumulateToMemory.p[Ty](
+  immarg i32,            ; opcode
+  %dx.types.MatrixRef *, ; matrix
+  [Ty] *,                ; groupshared T[M * N]
+  i32,                   ; Offset
+  i32,                   ; Stride
+  i32                    ; matrix layout
+  )
+```
+
+Accumulates a matrix to groupshared memory. This operation is only available for
+matrices with `MatrixUse::Accumulator` and `Wave` or `ThreadGroup` scope. Data
+conversions between opaque matrices and groupshared memory are defined in the
+[Conversions](#conversions) section below.
+
+The validator ensure that the group shared target memory is large enough for the
+write.

```

Comment:
Feels like there's a missing word here - "The validator will ensure" or "The validator must ensure" I think